xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -1483,7 +1483,7 @@ activitystream.event.deleteComment.rss.body=A comment has been deleted from the
 ### Deleting a page
 core.delete=Delete
 core.delete.title=Delete {0}
-core.delete.backlinksInfo=After deleting this page, the {0,choice,0#incoming links|1#incoming link|1<incoming links} will point to an empty page. To avoid this, you can select a new target.
+core.delete.backlinksMessage=After deleting this page, the {0} {1,choice,0#incoming links|1#incoming link|1<incoming links}{2} will point to an empty page. To avoid this, you can select a new target.
```

Comment:
Is it normal that there's a space between `{0}` and `{1`?

Also, didn't we have a link before? If so where is the key that was used? Cannot we reuse it?

Reply:
We just listed the backlinks [before](https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties#L5550), so there is no translation to reuse.
I removed the extra space, it wasn't indeed needed